cmd/docker/builder.go

```diff
@@ -117,33 +117,38 @@ func processBuilder(dockerCli command.Cli, cmd *cobra.Command, args, osargs []st
 	}
 
 	// overwrite the command path for this plugin using the alias name.
-	cmd.Annotations[pluginmanager.CommandAnnotationPluginCommandPath] = fmt.Sprintf("%s %s", cmd.CommandPath(), alias)
+	cmd.Annotations[pluginmanager.CommandAnnotationPluginCommandPath] =
+		strings.Join(append([]string{cmd.CommandPath()}, fwcmdpath...), " ")
```

Comment:
Oh! It doesn't like the wrapping here; think it's fine to keep it a single line;


```
64.44 cmd/docker/builder.go:120: File is not `gofumpt`-ed (gofumpt)
64.44 	cmd.Annotations[pluginmanager.CommandAnnotationPluginCommandPath] =
64.44 		strings.Join(append([]string{cmd.CommandPath()}, fwcmdpath...), " ")
```

Reply:
Weird. I only did this because it exceeded 80 characters.